scripts/upgrade

```diff
@@ -261,8 +261,8 @@ then
     ynh_replace_string --match_string="^\s*// installed plugins" --replace_string="&\n $installed_plugins" --target_file="$final_path/config/config.inc.php"
 
     # Update JavaScript dependencies
-    (cd "$final_path"
-    /usr/bin/php$phpversion -q ./bin/update.sh -v ?)
+    pushd "$final_path"
+    ./bin/update.sh --version="?" -y <<< ""
```

Comment:
Why is it done here and then on line 279?

Reply:
Still wonder why it's called twice...

Reply:
Not sure about the double call. But I added your fix and it seems to do the trick, thanks.